azurerm/resource_arm_bot_channel_msteams_test.go

```diff
@@ -0,0 +1,191 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/botservice/mgmt/2018-07-12/botservice"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func testAccAzureRMBotChannelMsTeams_basic(t *testing.T) {
+	// if ok := skipMsTeamsChannel(); ok {
+	// 	t.Skip("Skipping as one of `ARM_TEST_MsTeams_CLIENT_ID`, `ARM_TEST_MsTeams_CLIENT_SECRET`, or `ARM_TEST_MsTeams_VERIFICATION_TOKEN` was not specified")
+	// }
+	ri := tf.AccRandTimeInt()
+	config := testAccAzureRMBotChannelMsTeams_basicConfig(ri, testLocation())
+	resourceName := "azurerm_bot_channel_msteams.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMBotChannelMsTeamsDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMBotChannelMsTeamsExists(resourceName),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
```

Comment:
`ImportStateVerifyIgnore` is normally only called when a value isn't returned from the API and so we can't check whether it has been imported and set into state correctly. From my brief bit of testing, it does look like these values are returned from the api just fine. Did you see something different? If not, can we remove all the `ImportStateVerifyIgnore` from this file?

Reply:
This is a bit beyond my knowledge here, but happy to make whatever changes are needed to the PR. What exactly needs to be changed?

Reply:
This should be able to be removed since I saw these values being returned from the api. My concern was that it's been added so maybe you didn't see these values being returned from the api or did you copy this section in from another resource?